pkgs/games/badlion-client/default.nix

```
@@ -0,0 +1,32 @@
+{ appimageTools, fetchurl, lib }:
```

A general custom is to always have `lib` first, and then having things in the order of appearance in the code. So here, that would be:
```suggestion
{ lib, fetchurl, appimageTools }:
```